src/catkin_pkg/templates/metapackage.cmake.in

```diff
@@ -1,4 +1,4 @@
 cmake_minimum_required(VERSION 2.8.3)
-project(@name@)
+project(@name)
```

Comment:
Removing the trailing `@` character makes the file not expandable anymore by CMake `configure_file`. Please use the `configure_file` function (https://github.com/ros-infrastructure/catkin_pkg/blob/752c549237c1aaf04731967af4f46db20de71e9c/src/catkin_pkg/cmake.py#L50) instead.